### PR TITLE
Add Codesniffer file for WordPress Standards

### PR DESCRIPTION
**Pull request by dshanske**
This is the file @pfefferle set up and I use for checking against WordPress coding standards.

### CODE REVIEW

**Comment by snarfed** [comment]
cool! how do i run it? what do i need to install? could you add instructions to the readme?

**Comment by pfefferle** [comment]
* install [PHPCS](https://github.com/squizlabs/PHP_CodeSniffer)
* install and connect [WordPress-Coding-Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards)
* run in command line or install a plugin for your favourite editor (I use [linter-phpcs](https://github.com/AtomLinter/linter-phpcs) for Atom)

**Comment by dshanske** [comment]
What he said.

**Comment by snarfed** [comment]
great, thanks! happy to merge once that's in the readme.

**Comment by dshanske** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -85,3 +85,11 @@ To set up your local environment to run the unit tests:
     Time: 703 ms, Memory: 33.75Mb
     OK (1 test, 3 assertions)
     ```
+
+To set up PHPCodesniffer to test changes for adherance to WordPress Coding Standards
+
+1. install [PHPCS](https://github.com/squizlabs/PHP_CodeSniffer).
```
Comment (snarfed) [inline README.md]: minor point: in the future, you'll want to pay attention to details like capitalization (here and below), punctuation (period below), ```s around command lines, etc.

definitely not a big deal. we can clean these up later, any time.

**Comment by snarfed** [comment]
thanks! merging.

sad that phpcs isn't in homebrew, and that you have to install WordPress-Coding-Standards from git. ah well. definitely please do run them and fix whatever you find!

**Pull request merged**